src/convenience/frameworks.node.ts

```diff
@@ -49,12 +49,30 @@ const awsLambda = (event: any, _context: any, callback: any) => ({
     unauthorized: () => callback(null, { statusCode: 401 }),
 });
 
+/** Azure Functions */
+const azure = (context: any, req: any) => ({
```

Comment:
```suggestion
/** Azure Functions */
// deno-lint-ignore no-explicit-any
const azure = (context: any, req: any) => ({
```

Lint will be failing otherwise.

Reply:
Actually, no. `no-explicit-any` is already ignored for that entire file. Its the formatting that fails.

Reply:
Ah.